contexts/food-safety-certification/v1.json

```
@@ -0,0 +1,29 @@
+{
+  "@context": {
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+    "FoodSafetyCertificationCredential": "https://contexts.vcplayground.org/examples/food-safety-certification/vocab#FoodSafetyCertificationCredential",
+
+    "name": "https://schema.org/name",
+    "description": "https://schema.org/description",
+    "image": "https://schema.org/image",
+    "certification": "https://contexts.vcplayground.org/examples/food-safety-certification/vocab#certification",
+
+    "FoodSafetyCertification": {
+      "@id": "https://contexts.vcplayground.org/examples/food-safety-certification/vocab#FoodSafetyCertification",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+
+        "certificateId": "https://contexts.vcplayground.org/examples/food-safety-certification/vocab#certificateId",
+        "examDate": {
+          "@id": "https://contexts.vcplayground.org/examples/food-safety-certification/vocab#examDate",
+          "@type": "xsd:dateTime"
```

`xsd` is not defined.  Either define it somewhere or inline the full URL:
```suggestion
          "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
```

---

I missed that this was changed to `https`.  It likely needs to be `http`.  See related comment here: https://github.com/credential-handler/vc-examples/pull/18#discussion_r1238630667.